float64 front_leg_vel
float64 back_leg_vel
